### PR TITLE
Part1: Gradle wrapper 4.10.2, build gradle tools 3.2.1 and kotlin 1.2.71

### PR DESCRIPTION
**Pull request by nikitin-da**
https://docs.gradle.org/current/release-notes.html
https://developer.android.com/studio/releases/gradle-plugin
https://github.com/JetBrains/kotlin/releases

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
```
Comment (artem-zinnatullin) [inline gradle/wrapper/gradle-wrapper.properties]: I'm pretty sure `gradle-wrapper.jar` was changed and needs to be committed :)

**Comment by artem-zinnatullin** [comment]
```
> Task :storio-sqlite-annotations-processor-test:javaPreCompileReleaseUnitTest
Annotation processors must be explicitly declared now.  The following dependencies on the compile classpath are found to contain annotation processor.  Please add them to the testAnnotationProcessor configuration.
  - storio-sqlite-annotations-processor.jar (project :storio-sqlite-annotations-processor)
  - auto-service-1.0-rc4.jar (com.google.auto.service:auto-service:1.0-rc4)
  - auto-value-1.3.jar (com.google.auto.value:auto-value:1.3)
Alternatively, set android.defaultConfig.javaCompileOptions.annotationProcessorOptions.includeCompileClasspath = true to continue with previous behavior.  Note that this option is deprecated and will be removed in the future.
```

**Comment by artem-zinnatullin** [comment]
Uhhhh, damn AGP…

There is an option for android resources, but I don't see that for regular resources

Can you try enabling android one?

https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.TestOptions.UnitTestOptions.html

Like that:

```groovy
android {
  testOptions {
    unitTests {
     includeAndroidResources true
    }
  }
}
```

**Comment by artem-zinnatullin** [comment]
As an alternative, we can actually move it to android resources, like: `src/main|test/resources/raw/…`

**Comment by nikitin-da** [comment]
Closed via #897

**Pull request closed without merging**